### PR TITLE
[ng] src reorg of structural-directives example, and convert Jade to .md

### PR DESCRIPTION
**Pull request by chalin**
Contributes to #713

### CODE REVIEW

File: src/angular/guide/structural-directives.md [hunk 1]
```diff
@@ -0,0 +1,742 @@
...
+
+The drop down works properly.
+
+<img class="image-display" src='/resources/images/devguide/structural-directives/select-ngcontainer-anim.gif' alt="ngcontainer options work properly">
+
+The `<{{page.ng_container}}>` is a syntax element recognized by the Angular parser.
+It's not a directive, component, class, or interface.
+It's more like the curly braces in a Dart `if`-block:
+
+```dart
```
Comment (kwalrath) [inline src/angular/guide/structural-directives.md]: The formatting of this excerpt looks off (all the text is the same color, e.g.).

File: src/angular/guide/structural-directives.md [hunk 2]
```diff
@@ -0,0 +1,742 @@
...
+When the `condition` is false, the top (A) paragraph appears and the bottom (B) paragraph disappears.
+When the `condition` is true, the top (A) paragraph is removed and the bottom (B) paragraph appears.
+
+<img class="image-display" src='/resources/images/devguide/structural-directives/unless-anim.gif' alt="UnlessDirective in action">
+
+## Summary
+
+You can both try and download the source code for this guide in the <live-example></live-example>.
+
+Here is the source from the `lib` folder.
```
Comment (kwalrath) [inline src/angular/guide/structural-directives.md]: "from the lib folder" sounded weird to me. (Originally everything was right under lib and had no directories named in the tab.) Maybe from -> under?

File: src/angular/guide/structural-directives.md (hunk 1 above)
Comment (chalin) [inline src/angular/guide/structural-directives.md]: Fix, but I couldn't get the triple-tick markdown syntax to highlight the if statement as if it were Dart. I had to write it as HTML:
```
<code-example lang="dart">if (someCondition) {
  statement1;
  statement2;
  statement3;
}</code-example>
```

File: src/angular/guide/structural-directives.md (hunk 2 above)
Comment (chalin) [inline src/angular/guide/structural-directives.md]: Fixed

File: src/angular/guide/structural-directives.md (hunk 1 above)
Comment (kwalrath) [inline src/angular/guide/structural-directives.md]: That's weird. Why do you think that particular code snippet was special?

Comment (chalin) [inline src/angular/guide/structural-directives.md]: We don't have any other example of use of the triple-tick markdown syntax w/o a `<?code-excerpt?>` instruction before it. When that happens, Jekyll's default highlighter kicks in, and my guess is that it doesn't know how to handle Dart code.

**Pull request merged**